ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -408,8 +409,13 @@ private[kvutils] class TransactionCommitter(
       def foldInformeeParties(tx: Tx.Transaction, init: Boolean)(
           f: (Boolean, String) => Boolean
       ): Boolean =
-        tx.fold(init) { case (accum, (_, node)) =>
-          node.informeesOfNode.foldLeft(accum)(f)
+        tx.fold(init) {
+          case (accum, (_, node: Node.GenActionNode[_, _])) =>
+            node.informeesOfNode.foldLeft(accum)(f)
+          case (accum, (_, _: NodeRollback[_])) =>
+            // TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
and again same comment

Reply:
If the answer is the same for the 4 places, I will rather implement directly a method `informees` in GenTransaction class

Reply:
this is the kvutils version of `validateParties` above so same answer.

As I said, perfectly fine to do this in a separate PR. I don’t really like adding `informees`. This might be fine now but it seems easy to end up changing something later or adding a new function where rollback nodes shouldn’t be treated the same. Rollback nodes really don’t have informees.